utils/objectUtils.js

```diff
@@ -1,34 +1,45 @@
 /* eslint-disable no-process-env */
-const entityType = { SUITE: 'SUITE', TEST: 'STEP' },
+const entityType = { SUITE: 'SUITE', TEST: 'STEP' };
 
-    getStartLaunchObject = (options = {}) => ({
-        launch: process.env.RP_LAUNCH_NAME || options.launchname || 'Unit Tests',
-        tags: options.tags,
-        start_time: new Date().valueOf()
-    }),
+const getStartLaunchObject = (options = {}) => ({
+    launch: process.env.RP_LAUNCH || options.launch || 'Unit Tests',
+    description: options.description,
+    attributes: options.attributes,
+    startTime: new Date().valueOf()
+});
 
-    getTestStartObject = testTitle => ({
-        type: entityType.TEST,
-        name: testTitle || 'Custom Test title'
-    }),
+const getTestStartObject = testTitle => ({
+    type: entityType.TEST,
+    name: testTitle || 'Custom Test title',
+    description: testTitle || 'Custom description',
```

Comment:
Is it necessary to copy title in description? Also I think default description is redundant.